packages/core/src/icons/components/MinusOne.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default function SvgMinusOne(props) {
+  return (
+    <svg width="1em" height="1em" viewBox="0 0 30 30" fill="none" {...props}>
+      <circle cx={15} cy={15} r={14.5} fill="#fff" stroke="#B0B0B0" />
```

Comment:
這裡要把  fill 拿掉，不然如果背景是其他顏色，就會像下面的圖 😂 
circle stroke 的顏色可以找設計確認一下是不是固定的
```suggestion
      <circle cx={15} cy={15} r={14.5} fill="none" stroke="#B0B0B0" />
```

![image](https://user-images.githubusercontent.com/5608945/95726740-e2444d00-0cab-11eb-9ce8-f126966c8361.png)